### ISSUE TITLE
tutorial possibly outdated

### ISSUE THREAD

**Issue by chrisovergaauw** [cuba-labs/java-electron-tutorial#6]
Hi,

When following the tutorial to the letter, the application does not work initially. At least not for the first few steps underneath [vaadin ui at the speed of light](https://github.com/cuba-labs/java-electron-tutorial#vaadin-ui-at-the-speed-of-light)

running `gradle assemble jettyStart` at this points results in an app which only shows red text: Directory: /app/

I'll try to investigate it and make a PR when I find the issue. It is a bit of a bummer as a first introduction to the framework though.

Kind Regards,
Chris

**Comment by jreznot** [comment]
I'm going to update the tutorial with the latest Vaadin 12 a little bit later

**Comment by chrisovergaauw** [comment]
That would be greatly appreciated!

**Comment by LegacyKing** [comment]
Ditto (+1)  Trying to follow the tutorial and I hit a wall at the section

> At the moment, we've got all the pieces of the puzzle sitting right there on the table. Now all we have to do is put them in the right order.
> 
> Bring all together

Any clarity on what files are being edited and where after the initial steps would be appreciated (And if we can validate against the Java 11 even better). 

Thanks!

**Comment by akumthek** [comment]
Will appreciate an update. In the meantime i will try nodejs child process approach which seems simpler than websockets

**Comment by ghost** [comment]
yes. I tried many times and it does not work. Hope that the tutorial will be updated soon. Thanks for your effort.

**Comment by bkstorm** [comment]
It works now.
